billing.py

```diff
@@ -1,27 +1,19 @@
-from flask import Flask, jsonify, request, session, Blueprint
-from flask_login import LoginManager, login_required, \
-    current_user, UserMixin
-# import json
-# from flask_sqlalchemy import SQLAlchemy
-from flask_cors import CORS, cross_origin
-from flask_migrate import Migrate
-# import flask_excel as excel
+from flask import jsonify, request, Blueprint
+from flask_login import login_required, \
+    current_user
+from flask_cors import cross_origin
 from flask.ext.elasticsearch import Elasticsearch
 from flask import redirect
-# import ast
 from decimal import Decimal
-# import copy
-
 import os
-from models import Billing, db
-from utility import get_compute_costs, get_storage_costs, create_analysis_costs_json, create_storage_costs_json
+from models import Billing
+from utility import get_compute_costs, get_storage_costs,\
+    create_analysis_costs_json, create_storage_costs_json
 import datetime
 import calendar
 import click
```

Comment:
Hi Carlos, unrelated to this review. Are people using more `click` or more `argparse` in our group? What do you prefer?

Reply:
I think it depends on the context? I personally use `argparse`. However the person that originally wrote the billing portion of this decided to use `click`. So to not disturb the functionality i decided to not move `click` to `argparse`